pkg/kubelet/container_gc.go

```diff
@@ -79,6 +86,12 @@ type containerGCInfo struct {
 
 	// Creation time for the container.
 	createTime time.Time
+
+	// Pod name
+	podName string
```

Comment:
Also change the name of the variable. Don't call it a name if it includes a namespace. :)